apps/client/src/app/components/admin-market-data/asset-profile-dialog/asset-profile-dialog.html

```diff
@@ -215,6 +218,12 @@ <h1 class="flex-grow-1 m-0" mat-dialog-title>
         <input formControlName="name" matInput type="text" />
       </mat-form-field>
     </div>
+    <div class="mt-3">
```

Comment:
As `url` it is not that important, can you move it down just before the `comment` field?

Reply:
Done